README_zh-CN.md

```diff
@@ -61,192 +41,123 @@ MMCV 提供了如下众多功能：
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu111/torch1.8.0/index.html</code></pre> </details> </td>
-      <td align="left"> </td>
-      <td align="left"> </td>
-      <td align="left"> </td>
+      <td align="left">11.1</td><td align="left"></td><td align="left">1.3.16</td>
+      <td align="left">1.3.9</td><td align="left">1.3.0</td><td align="left"></td>
+      <td align="left"></td><td align="left"></td>
     </tr>
     <tr>
-      <td align="left">11.0</td>
-      <td align="left"> </td>
-      <td align="left"> </td>
-      <td align="left"> </td>
-      <td align="left"> </td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu110/torch1.7.0/index.html</code></pre> </details> </td>
-      <td align="left"> </td>
-      <td align="left"> </td>
+      <td align="left">11.0</td><td align="left"></td><td align="left"></td>
+      <td align="left"></td><td align="left"></td><td align="left">1.2.0</td>
+      <td align="left"></td><td align="left"></td>
     </tr>
     <tr>
-      <td align="left">10.2</td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.11.0/index.html</code></pre> </details></td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.10.0/index.html</code></pre> </details> </td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.9.0/index.html</code></pre> </details> </td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.8.0/index.html</code></pre> </details> </td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.7.0/index.html</code></pre> </details> </td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.6.0/index.html</code></pre> </details> </td>
-      <td align="left"><details><summary> 安装 </summary><pre><code>pip install mmcv-full=={mmcv_version} -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.5.0/index.html</code></pre> </details> </td>
+      <td align="left">10.2</td><td align="left">1.4.7 (Linux only)</td><td align="left">1.3.16</td>
+      <td align="left">1.3.9</td><td align="left">1.3.0</td><td align="left">1.2.0</td>
+      <td align="left">1.0.5</td><td align="left">1.0.0</td>
```

Comment:
Can we make Linux only in the same line? 
![image](https://user-images.githubusercontent.com/57566630/177492693-b990bfb4-6384-4eac-b234-62d76136ba22.png)